networking/network_observability/installing-operators.adoc

```diff
@@ -5,7 +5,7 @@ include::_attributes/common-attributes.adoc[]
 :context: network_observability
 
 toc::[]
-Installing Loki is a prerequisite for using the Network Observability Operator. It is recommended to install Loki using the Loki Operator; therefore, these steps are documented below prior to the Network Observability Operator installation.
+Installing Loki is a recommended prerequisite for using the Network Observability Operator. Therefore, these steps are documented below prior to the Network Observability Operator installation. You can choose to use xref:../../networking/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki], but there are some considerations for doing this. 
```

Comment:
IBM SG contains a complete section on claims and recommendations. Best to avoid the term, IMO. 

![Screenshot from 2023-08-31 12-41-00](https://github.com/openshift/openshift-docs/assets/57954076/01211e71-55c3-490a-93dc-dfa39c7ffd29)

Reply:
> there are some considerations for doing this

there are some limitations, described in the following section, that you should consider

Reply:
Ok, @mburke5678 can I say: You can choose to use xref:../../networking/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki], but there are some considerations for doing this, described in the previous linked section.

@dfitzmau I don't think this is an _unsubstantiated_ claim or recommendation because the Operator exports data to Loki, its just part of the way that customers use the product. Without Loki, there are some parts of the web console plugin that aren't available so its really the complete experience to use Loki too. I guess that's why we phrase it as recommendation. Does that make sense, or still seem like an overreaching claim to you?

Reply:
Thanks, @skrthomas , for clarifying. Yes, this makes sense and I see your point. The recommendation is on the minor side of things as against say configuring some security policies (which would be more complex recommendations).